client/my-sites/checkout/upsell-nudge/white-glove/index.jsx

```diff
@@ -0,0 +1,114 @@
+/**
+ * External dependencies
+ */
+import React, { PureComponent } from 'react';
+import formatCurrency from '@automattic/format-currency';
+
+/**
+ * Internal dependencies
+ */
+import { CompactCard, Button } from '@automattic/components';
+import DocumentHead from 'components/data/document-head';
+import PageViewTracker from 'lib/analytics/page-view-tracker';
+import WordPressLogo from 'components/wordpress-logo';
+import Checklist from './checklist';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+/**
+ * Image dependencies
+ */
+
+export class WhiteGlove extends PureComponent {
+	render() {
+		const title = 'Checkout ‹ White Glove';
+
+		return (
+			<>
+				<PageViewTracker path="/checkout/offer-white-glove/:site" title={ title } />
```

Comment:
Just noting that this is different from the actual URL path which would be `/checkout/:site/offer-white-glove`.
It looks like the other offer pages don't use either order consistently.